src/v2/Apps/Onboarding/Components/OnboardingGene.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { OnboardingGene_gene } from "v2/__generated__/OnboardingGene_gene.graphql"
+import { OnboardingGeneQuery } from "v2/__generated__/OnboardingGeneQuery.graphql"
+import { OnboardingLoadingCollection } from "v2/Apps/Onboarding/Components/OnboardingLoadingCollection"
+import { ArtworkGridItemFragmentContainer } from "v2/Components/Artwork/GridItem"
+
+import { Masonry } from "v2/Components/Masonry"
+import { extractNodes } from "v2/Utils/extractNodes"
+
+interface OnboardingGeneProps {
+  gene: OnboardingGene_gene
+}
+
+const OnboardingGene: FC<OnboardingGeneProps> = ({ gene }) => {
+  const artworks = extractNodes(gene.artworks)
+
+  return (
+    <>
```

Comment:
Looks good. Let's get the rest of this in here: the title and the follow button: https://github.com/artsy/force/blob/main/src/v2/Components/FollowButton/FollowGeneButton.tsx